appserver/pom.xml

```diff
@@ -971,8 +971,8 @@
                             </artifact>
                             <nonFinal>false</nonFinal>
                             <jarType>api</jarType>
-                            <specVersion>2.3</specVersion>
-                            <specImplVersion>2.3.1</specImplVersion>
+                            <specVersion>3.1</specVersion>
+                            <specImplVersion>3.1.1</specImplVersion>
```

Comment:
Why not use `${jakarta.pages-api.version}` instead of hard-coded value?

Reply:
> Why not use `${jakarta.pages-api.version}` instead of hard-coded value?

I will update.